lib/controllers/admin.ts

```diff
@@ -523,5 +523,19 @@ export default (crowi: Crowi) => {
     return res.json(ApiResponse.success({ message: 'Now re-building backlinks ... this takes a while.' }))
   }
 
+  actions.api.user.edit = async function (req: Request, res: Response) {
+    const { name, emailToBeChanged, currentEmail } = req.form.userEditForm
+
+    try {
+      const user = await User.findUserByEmail(currentEmail)
```

Comment:
💡 Should find user by userId (To do so, pass target user id via hidden form element)

Reply:
❌ Check if existing user already has an email to be updated?